File: tests/fixtures/languages/typescript/src/example_domain/example_services/example_policy.ts
import { ExampleRecord } from '../example_model/example_record';

export class ExamplePolicy {
    example_allows(example_record: ExampleRecord): boolean {
        return exampleAllowed(example_record);
    }
}

export function exampleAllowed(example_record) {
    return !example_record.example_active;
}

File: tests/fixtures/languages/python/src/example_domain/example_services/example_policy.py
from example_domain.example_model.example_record import ExampleRecord


class ExamplePolicy:
    def example_allows(self, example_record: ExampleRecord) -> bool:
        return example_allowed(example_record)


def example_allowed(example_record: ExampleRecord) -> bool:
    return not example_record.example_active
